mux_test.go

```diff
@@ -35,8 +35,10 @@ type routeTest struct {
 	path           string            // the expected path of the match
 	pathTemplate   string            // the expected path template to match
 	hostTemplate   string            // the expected host template to match
+	additionalPath string            // additional path to add to route (and parse) during test
```

Comment:
Agreed, I don't think it's necessary at all. Since calling `Path` panics right away, that can just be done as its own test. Not all the tests need to follow the exact same structure and go through `testRoute`.

This way `shouldPanic` can also be removed.